sdk/cosmos/azure-cosmos/azure/cosmos/user.py

```diff
@@ -117,15 +117,16 @@ def query_permissions(
         max_item_count=None,
         **kwargs
     ):
-        # type: (str, Optional[List[str]], Optional[int], Any) -> Iterable[Dict[str, Any]]
+        # type: (str, Optional[List[Dict[str, Any]]], Optional[int], Any) -> Iterable[Dict[str, Any]]
         """Return all permissions matching the given `query`.
 
-        :param query: The Azure Cosmos DB SQL query to execute.
+        :param str query: The Azure Cosmos DB SQL query to execute.
         :param parameters: Optional array of parameters to the query. Ignored if no query is provided.
-        :param max_item_count: Max number of permissions to be returned in the enumeration operation.
+        :type parameters: List[Dict[str, Any]]
```

Comment:
Are we sure this is a `List[Dict]]`?
The docstring describes it as an `array of parameters`, whereas the usage below on line 138 treats it as a straight `Dict`...

Reply:
we package the parameters into a dictionary along with the query, but yes it is a list of dictionaries

Reply:
![image](https://github.com/Azure/azure-sdk-for-python/assets/30335873/27167b75-ba9f-4049-ac3e-ec392591c844)